### ISSUE TITLE
`Ctrl` `del` deletes file even when focused on document editor

### ISSUE THREAD

**Issue by BenjaminPhillips22** [vkbo/novelWriter#629]
`Ctrl` `del` deletes file even when focused on document editor, instead of deleting the word after the cursor like expected.

Steps to recreate issue;

Open Project.
Press `Alt` `1` to focus on project tree.
Select file to write in.
Press `Alt` `2` to focus on document editor.
Write in the document editor.
Press `Ctrl` `del`.

**Comment by vkbo** [comment]
`Ctrl`+`Del` is a menu shortcut for the delete item entry. The QTextEdit widget does not support `Ctrl`+`Del` (or `Ctrl`+`Backspace`). This is not a bug.

It's a potential feature suggestion though.

**Comment by vkbo** [comment]
That said, it's probably a good idea to block the key press when the editor has focus, even if the menu entry itself works.

**Comment by vkbo** [comment]
The focus requirement has been merged to `main`. I'll leave this open for the other two features.

**Comment by vkbo** [comment]
So, I did some testing, and it turns out that `Ctrl+Backspace` and `Ctrl+Del` actually do work. However, they aren't documented in the [Qt documentation](https://doc.qt.io/qt-5/qtextedit.html#editing-key-bindings). I found out by disabling the existing `Ctrl+Del` shortcut currently used for deleting files in the project tree.

I'll free up this shrotcut again by changing the delete item shortcut to `Ctrl+Shift+Del` instead. It should be available in the next testing release for version 1.2. I released 1.2 Beta 1 today, but didn't check this until after. The fix is in PR #664.
